src/Routes.js

```diff
@@ -1,82 +1,61 @@
 //> React
 // Contains all the functionality necessary to define React components
-import React from 'react';
+import React from "react";
 // DOM bindings for React Router
-import { Route, Switch, Redirect } from 'react-router-dom';
+import { Route, Switch } from "react-router-dom";
 
 //> Components
 /**
  * HomePage: A basic template page
  */
-import {
-  HomePage,
-  About,
-  Privacy,
-  Cancellation,
-  Shipping,
-  Terms,
-} from './components/pages';
+import { HomePage, MessagePage } from "./components/pages";
 
 class Routes extends React.Component {
   render() {
     return (
       <Switch>
         <Route
-        exact
-        path='/about'
-        component={(props) => <About
-        {...this.props}
-        />
-        }
+          exact
+          path="/"
+          render={(props) => (
+            <HomePage client={this.props.client} {...this.props} />
+          )}
         />
         <Route
-        exact
-        path='/privacy'
-        component={(props) => <Privacy
-        {...this.props}
-        />
-        }
+          exact
+          path="/about"
+          render={(props) => <MessagePage {...this.props} />}
         />
         <Route
-        exact
-        path='/terms'
-        component={(props) => <Terms
-        {...this.props}
-        />
-        }
+          exact
+          path="/agb"
+          render={(props) => <MessagePage {...this.props} />}
         />
         <Route
-        exact
-        path='/shipping'
-        component={(props) => <Shipping
-        {...this.props}
-        />
-        }
+          exact
+          path="/shipping"
+          render={(props) => <MessagePage {...this.props} />}
         />
         <Route
-        exact
-        path='/cancellation'
-        component={(props) => <Cancellation
-        {...this.props}
-        />
-        }
+          exact
+          path="/widerruf"
```

Comment:
Path name could be changed to "cancellation", to be in the same language as the other pathnames.